src/btchip_helpers.c

```diff
@@ -24,6 +24,20 @@ const unsigned char TRANSACTION_OUTPUT_SCRIPT_PRE[] = {
 const unsigned char TRANSACTION_OUTPUT_SCRIPT_POST[] = {
     0x88, 0xAC}; // OP_EQUALVERIFY, OP_CHECKSIG
 
+const unsigned char ZEN_OUTPUT_SCRIPT_PRE[] = {
+    0x3F, 0x76, 0xA9,
+    0x14}; // script length, OP_DUP, OP_HASH160, address length
+const unsigned char ZEN_OUTPUT_SCRIPT_POST[] = {
+    0x88, 0xAC, // OP_EQUALVERIFY, OP_CHECKSIG
+    0x20, 0x9e, 0xc9, 0x84, 0x5a, 0xcb, 0x02, 0xfa, 0xb2, 0x4e, 0x1c, 0x03, 0x68,
+    0xb3, 0xb5, 0x17, 0xc1, 0xa4, 0x48, 0x8f, 0xba, 0x97, 0xf0, 0xe3, 0x45, 0x9a, 0xc0, 0x53,
+    0xea, 0x01, 0x00, 0x00, 0x00, // ParamHash
+    0x03, // Push 3 bytes to stack to make ParamHeight line up properly
+    0xc0, 0x1f, 0x02, //ParamHeight (139200) -> hex -> endianness swapped
```

Comment:
From BIP115, I see that the block height parameter should refer to a block at most 52596 block deep in the chain
So if you hardcode this value, one day  it will become an invalid one. 
The zencash chain has a block time of 2min31s, so the blockheight must be updated at least every ~90 days